app/assets/javascripts/sephora_style_guide/application.js

```diff
@@ -1,4 +1,5 @@
 //= require 'jquery'
 //= require_self
+//= require './transition'
```

Comment:
There is already a require_tree right below so you shouldn't need this. Also that whole file is commented out?